core/src/main/scala/io/finch/Input.scala

```diff
@@ -19,4 +56,26 @@ object Input {
   def apply(req: Request): Input = Input(req, req.path.split("/").toList.drop(1))
 
   implicit val inputEq: Eq[Input] = Eq.fromUniversalEquals
+
+  def get(path: String): Input = Input(Request(path))
```

Comment:
I don't think we need these methods given that `(String, String)*` defaults to empty list.